src/Microsoft.ML.Transforms/MutualInformationFeatureSelection.cs

```diff
@@ -105,13 +105,13 @@ internal sealed class Options : TransformInputBase
 
         private IHost _host;
         private readonly (string outputColumnName, string inputColumnName)[] _columns;
-        private readonly string _labelColumn;
+        private readonly string _labelColumnName;
         private readonly int _slotsInOutput;
         private readonly int _numBins;
 
         /// <include file='doc.xml' path='doc/members/member[@name="MutualInformationFeatureSelection"]/*' />
         /// <param name="env">The environment to use.</param>
-        /// <param name="labelColumn">Name of the column to use for labels.</param>
+        /// <param name="labelColumnName">Name of the column to use for labels.</param>
```

Comment:
Will this be a breaking change for ML.NET users?

Reply:
It will, re-compilation will do the trick. Already trained models will be fine. This is a missed change from previous push where LabelColumn was renamed LabelColumnName, WeightColumn was renamed to WeightColumnName and so on.Currently it acts as a technical debt in the sense that it needs a special treatment in NimbusML

---
In reply to: [314471870](https://github.com/dotnet/machinelearning/pull/4117#discussion_r314471870) [](ancestors = 314471870)

Reply:
This is an internal API, so it isn't breaking external users.

Reply:
Awesome. Thanks for checking.

Reply:
could one of you sign off :)